app/services/datum.service.go

```diff
@@ -0,0 +1,53 @@
+package services
+
```

Comment:
Nice, we will rip and replace the function implementations, but do have a look at how

Reply:
At how what 😂

Reply:
Oops lol, check how the authentication works on other endpoints in the handlers to see how we handle the multi-tenant database.